api/src/test/java/org/openmrs/module/registrationcore/api/RegistrationCoreServiceTest.java

```diff
@@ -282,7 +284,15 @@ public void registerPatient_shouldSucceedIfBiometricsSuppliedAndEngineEnabled()
         assertEquals("ISO", fingerprint.getFormat());
         assertEquals("xxxyyyzzz", fingerprint.getTemplate());
     }
-
+    
+    @Test
+    public void  registerPatient_shouldAlsoConsumeUuidAsValueForIdentifierSourceIdGP() {
+    	String sourceUuid = Context.getService(IdentifierSourceService.class).getIdentifierSource(1).getUuid();
+    	adminService.saveGlobalProperty(new GlobalProperty(RegistrationCoreConstants.GP_OPENMRS_IDENTIFIER_SOURCE_ID, sourceUuid));
+		Location location = locationService.getLocation(2);
+	    service.registerPatient(createBasicPatient(), null, null, location);
+    }
```

Comment:
This test doesn't assert anything.

Reply:
Ooohh, I just forgot to do some assertion